index.css

```diff
@@ -0,0 +1,540 @@
+  display: grid;
+  place-items: center;
+}
+
+/* until now, only donate.html */
+.flex {
+  display: flex;
+  gap: 1rem;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  font-family: 'Montserrat', sans-serif;
+}
+#desktop-header {
+  display: none;
+}
+#mobile-header {
+  /* background-color: var(--yellow); */
+  background-color: var(--yellow);
+  position: fixed;
+  width: 100%;
+  top: 0;
+  height: var(--mobile-nav-height);
+  z-index: 2;
+  transition: all 0.5s ease-in-out;
+}
+
+.mobile-navigation {
+  margin: 0 1rem;
+}
+.button {
+  color: var(--white);
+  text-decoration: none;
+  background-color: var(--black);
+  border-radius: 5px;
+  padding: 0.3rem 0.6rem;
+  font-family: 'Roboto', sans-serif;
+  cursor: pointer;
+}
+
+.button:hover,
+.button:focus {
+  background-color: var(--red);
+}
+
+/* -----burger menu------ */
+
+.hamburger {
+  width: 2rem;
+  height: 2rem;
+  z-index: 3;
+  cursor: pointer;
+}
+
+.hamburger .top-bun {
+  position: relative;
+  width: 2rem;
+  height: 2px;
+  /* background-color: var(--black); */
+  background-color: var(--red);
+  margin-top: 8px;
+  transition: all 0.3s ease-in-out;
+}
+
+.hamburger .bottom-bun {
+  position: relative;
+  width: 2rem;
+  height: 2px;
+
+  /* background-color: var(--black); */
+  background-color: var(--red);
+  margin-top: 8px;
+  transition: all 0.3s ease-in-out;
+}
+
+.nav {
+  position: fixed;
+  width: 100%;
+  height: 100vh;
+  background-color: var(--yellow);
+  top: -100%;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  overflow: hidden;
+  transition: all 0.3s ease-in-out;
+  transform: scale(0);
+  z-index: 2;
+}
+
+.nav-wrapper {
+  position: relative;
+  overflow: hidden;
+  overflow-y: auto;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+nav {
+  text-align: center;
+}
+
+nav ul {
+  text-transform: uppercase;
+  display: grid;
+  place-items: center;
+}
+
+nav ul li a {
+  font-family: 'Questrial', sans-serif;
+  position: relative;
+  text-decoration: none;
+  color: var(--black);
+  font-size: 25px;
+  display: inline-block;
+  margin-top: 1.25em;
+  transition: color 0.2s ease-in-out;
+  letter-spacing: 1px;
+}
+
+nav ul li a:hover,
+a:focus {
+  color: var(--red);
+}
+
+#toggle:checked + .hamburger .top-bun {
+  transform: rotate(-45deg);
+  width: 2em;
+  margin-top: 25px;
+  /* background-color: var(--red); */
+}
+#toggle:checked + .hamburger .bottom-bun {
+  transform: rotate(45deg);
+  margin-top: -2px;
+  /* background-color: var(--red); */
+}
+
+#toggle:checked + .hamburger + .nav {
+  top: 0;
+  transform: scale(1);
+}
+
+#toggle {
+  display: none;
+}
+
+/* ------main------ */
+.img-blocks:first-child {
+  height: calc(550px - 5rem);
+}
+.img-blocks {
+  min-height: 550px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  /* change attachment to fixed for desktop */
+  background-attachment: scroll;
+  display: grid;
```

Comment:
this grid made a huge gap between the title and the read more. I think if they come closer looks better but let me know if you have any other idea about it

Reply:
fixed